src/Raven.Studio/typescript/components/pages/database/settings/clientConfiguration/ClientDatabaseConfiguration.tsx

```diff
@@ -0,0 +1,321 @@
+                                    <FormSwitch
+                                        control={control}
+                                        name="loadBalancerSeedEnabled"
+                                        color="primary"
+                                        disabled={!useSessionContextEnabled || !overrideConfig}
+                                        label="Seed"
+                                        afterChange={(event) =>
+                                            !event.target.checked && setValue("loadBalancerSeedValue", null)
+                                        }
+                                    >
+                                        Seed
+                                    </FormSwitch>
+                                </Col>
+                                <Col>
+                                    <FormInput
+                                        type="number"
+                                        control={control}
+                                        name="loadBalancerSeedValue"
+                                        placeholder="Enter seed number"
+                                        disabled={!loadBalancerSeedEnabled || !overrideConfig}
+                                    />
+                                </Col>
+                            </Row>
+                        </Col>
+                        {globalConfig && (
+                            <>
+                                <Col>
+                                    <Input value={globalConfig.loadBalancerSeedValue || undefined} disabled />
+                                </Col>
+                                <Col></Col>
+                            </>
+                        )}
+                    </Row>
+
+                    <Row className="p-2">
+                        <Col>
+                            <FormCheckbox
+                                control={control}
+                                name="readBalanceBehaviorEnabled"
+                                afterChange={(event) =>
+                                    event.target.checked
+                                        ? setValue("readBalanceBehaviorValue", "None")
+                                        : setValue("readBalanceBehaviorValue", null)
+                                }
+                                disabled={!overrideConfig}
+                            >
+                                Read balance behavior
+                            </FormCheckbox>
+                        </Col>
+                        <Col>
+                            <FormSelect
+                                control={control}
+                                name="readBalanceBehaviorValue"
+                                disabled={!readBalanceBehaviorEnabled || !overrideConfig}
+                            >
+                                <FormSelectOption<ReadBalanceBehavior> label="None" value="None" />
```

Comment:
what about empty option? (-select-)

Reply:
It's reactstrap so I guess you'd need to add an option
`<option label="(-select-)" value={null} />`

or I can add e.g. `showEmptyOption` props to `<FormSelect />`